source/globalCommands.py

```diff
@@ -702,6 +702,32 @@ def script_toggleReportTableCellCoords(self,gesture):
 			config.conf["documentFormatting"]["reportTableCellCoords"]=True
 		ui.message(state)
 
+	@script(
+		# Translators: Input help mode message for toggle report cell borders command.
+		description=_("Cycles through  the cell border reporting settings"),
+		category=SCRCAT_DOCUMENTFORMATTING,
+	)
+	def script_toggleReportCellBorders(self, gesture):
+		if (
+			not config.conf["documentFormatting"]["reportBorderStyle"] and not
+			config.conf["documentFormatting"]["reportBorderColor"]

```

Comment:
Just to improve readability

```suggestion
			not config.conf["documentFormatting"]["reportBorderStyle"] and
			not config.conf["documentFormatting"]["reportBorderColor"]
```